devtools/cli/src/main/java/io/quarkus/cli/Build.java

```diff
@@ -1,57 +1,73 @@
 package io.quarkus.cli;
 
-import java.nio.file.Path;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.concurrent.Callable;
 
-import io.quarkus.cli.core.BaseSubCommand;
-import io.quarkus.cli.core.BuildsystemCommand;
-import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.cli.options.OutputOptions;
 import picocli.CommandLine;
+import picocli.CommandLine.Mixin;
 
-@CommandLine.Command(name = "build", mixinStandardHelpOptions = false, description = "Build your quarkus project")
-public class Build extends BaseSubCommand implements BuildsystemCommand {
+@CommandLine.Command(name = "build", mixinStandardHelpOptions = false, description = "Build the current project")
+public class Build implements Callable<Integer> {
 
-    @CommandLine.Option(names = { "-n",
-            "--native" }, order = 4, description = "Build native executable.")
+    @Mixin
+    OutputOptions output;
+
+    @CommandLine.Option(order = 3, names = { "-c",
+            "--clean" }, description = "Clean")
+    boolean clean = false;
```

Comment:
I would almost say that this should be `true` by default. I know that it's a special goal/task in maven/gradle but from my exp it's always good to start the build with `mvn clean ...`. Or at least it would help if it would be possible to say _"always clean my project during build so that I can do `quarkus build`"_ once and the option would the persisted somewhere...

Reply:
interesting thought. I think I could make this a toggle-able thing... (the --clean --no-clean variety? )